src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/updatelistingrequirements/UpdateListingRequirementsAboutToSubmitHandler.java

```diff
@@ -90,4 +104,24 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
                 .build());
         return callbackResponse;
     }
+
+    private boolean updateHearingDuration(SscsCaseData sscsCaseData, Optional<CaseDetails<SscsCaseData>> sscsCaseDataBefore) {
+        if (nonNull(sscsCaseData.getSchedulingAndListingFields().getOverrideFields().getDuration()) || isNull(sscsCaseData.getSchedulingAndListingFields().getDefaultListingValues())) {
+            return false;
+        }
+        boolean channelUpdated = false;
+        if (sscsCaseDataBefore.isPresent()) {
+            OverrideFields overrideFieldsBefore = sscsCaseDataBefore.get().getCaseData().getSchedulingAndListingFields().getOverrideFields();
+            HearingChannel channelBefore = nonNull(overrideFieldsBefore) ? overrideFieldsBefore.getAppellantHearingChannel() : null;
+            HearingChannel channelCurrent = sscsCaseData.getSchedulingAndListingFields().getOverrideFields().getAppellantHearingChannel();
+            channelUpdated = !Objects.equals(channelBefore, channelCurrent);
+        }
+        HearingInterpreter appellantInterpreter = sscsCaseData.getSchedulingAndListingFields().getOverrideFields().getAppellantInterpreter();
+        Optional<HearingOptions> hearingOptions = Optional.ofNullable(sscsCaseData.getAppeal().getHearingOptions());
+        String caseInterpreter = hearingOptions.isPresent() && nonNull(hearingOptions.get().getLanguageInterpreter())

```

Comment:
why create an optional when we need to do .get()? we can do nullCheck with isNull or nonNull